docs/learn/learn-governance.md

```diff
@@ -36,6 +36,52 @@ together to administrate a network upgrade decision. No matter whether the propo
 the public (token holders) or the [Council](#council), it finally will have to go through a vote on
 a referendum to let all holders, weighted by stake, make the decision.
 
+## Summary
+
+The figure below shows an overview of Governance V1 with main players and different paths from
+submitting proposal to voting on referenda.
+
+![gov1-overview](../assets/gov1-overview.png)
+
+The public (i.e. token holders) can submit a proposal that will end in the proposal queue. Here,
+proposals are [endorsed](#endorsing-proposals), and the ones that get the most support will climb in
+the queue. The proposal at the top of the queue will become a
+[Public Referendum](#public-referenda). See in the figure the vertical green path ending with the
+proposals inside a cylinder (i.e. the proposal queue). The proposal with 11 endorsements is at the
+top of the queue, ready to enter a referendum.
+
+The public can also submit a [treasury proposal](./learn-treasury.md#creating-a-treasury-proposal),
+which must be evaluated by the [Council](#council) through a motion. If the Council motion passes,
+the treasury proposal can be directly executed or go to the external queue, which will be voted on
+through a [Council Referendum](#council-referenda). See the figure's green horizontal path from the
+Public (green) to the Council (yellow). Treasury proposals and Council proposals can be directly
+executed (horizontal yellow arrows) or go to the external queue, where they will become a referendum
+(vertical yellow arrows).
+
+Note that the external queue always consists of
+[a single proposal](https://github.com/paritytech/substrate/blob/f4a2e84ee5974b219f2a03cd195105060c41e3cd/frame/democracy/src/lib.rs#LL29C8-L31C4).
+A proposal in the external queue can be fast-tracked by the
+[Technical Committee](#technical-committee) (light blue). The fast track can contain as many
+proposals as possible (also called emergency proposals) that can be voted on simultaneously with
+Council or Public Proposals. See in the figure the yellow circle (i.e. Council Proposal) exiting the
```

Comment:
```suggestion
either the Council or the Public. See in the figure the yellow circle (i.e. Council Proposal) exiting the
```

I did not quite understand this - maybe you meant what I suggested?  Disregard if this changes the meaning.

Reply:
fast-tracked proposals can be voted on simultaneously with either Council or Public proposals because the alternating timeline did not allow you to vote on Public and Council simultaneously. Does this help?